gita/__main__.py

```diff
@@ -397,6 +406,10 @@ def main(argv=None):
                 "and create hierarchical groups based on folder structure.")
     xgroup.add_argument('-b', '--bare', action='store_true',
             help="add bare repo(s)")
+    xgroup.add_argument('-g','--group', 
+                    nargs='?', 
```

Comment:
maybe omit this line, so that the user has to specify a group

Reply:
agreed